db/schema.rb

```diff
@@ -52,6 +52,28 @@
     t.index ["type"], name: "index_badges_on_type", unique: true
   end
 
+  create_table "contributor_team_memberships", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
```

Comment:
I know nothing about this repo, but these changes seem unrelated

Reply:
Yeah, I sometimes get that. Removed.